adapters/repos/db/index_queue.go

```diff
@@ -127,22 +142,40 @@ func NewIndexQueue(
 	if opts.Logger == nil {
 		opts.Logger = logrus.New()
 	}
-	opts.Logger = opts.Logger.WithField("component", "index_queue")
+	opts.Logger = opts.Logger.
+		WithField("component", "index_queue").
+		WithField("shard_id", shardID)
 
 	if opts.BatchSize == 0 {
 		opts.BatchSize = 1000
 	}
+	if v, _ := strconv.Atoi(os.Getenv("ASYNC_BATCH_SIZE")); v > 0 {
+		opts.BatchSize = v
+	}
```

Comment:
do we provide any `opts.BatchSize` default values, just in case somebody mistakenly provides `ASYNC_BATCH_SIZE` that is unparsebale to number?

Reply:
Yes the default is 1000, it is assigned on L.150